force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +122,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+        Integer countVowels = 0;
+        List<String> vowels = new List<String>{'a', 'e', 'i', 'o', 'u'};
+        List<String> wordSplitted = str.split('');
+
+        for(String letter : wordSplitted){
+            if(vowels.contains(letter)){
+                countVowels += 1;
+            }else{
+                continue;
+            }
```

Comment:
There's no need for else or continue, they do nothing. When the code gets compiled, these are basically ignored because they are two instructions at the end of a loop with no instructions after them.
```suggestion
        for(String letter : letters) {
            if(vowels.contains(letter)) {
                countVowels += 1;
            }
```